doc/source/user_guide/timeseries.rst

```diff
@@ -853,6 +853,9 @@ savings time. However, all :class:`DateOffset` subclasses that are an hour or sm
 The basic :class:`DateOffset` acts similar to ``dateutil.relativedelta`` (`relativedelta documentation`_)
 that shifts a date time by the corresponding calendar duration specified. The
 arithmetic operator (``+``) can be used to perform the shift.
+Multiplication (*) and subtraction are also supported. Besides arithmetic operations,
```

Comment:
Probably better this note and example below are put in a separate section. This part looks to specifically demonstrate the difference of timezone behavior between `DateOffset` and `Timedelta`

Reply:
Actually, can you just move this example directly in the `DateOffset` docstring? It tends to get more viability.

Reply:
Sounds good. It's moved.